create-project

```diff
@@ -38,8 +38,11 @@ def create_site(domain):
         "image": "prod-comms.docker-registry.canonical.com/" + name,
     }
 
+    data["production"] = {}
+    data["production"]["replicas"] = 5
```

Comment:
If it's mandatory, don't we explicitly *not* want to set it here?

Reply:
Oh no I misunderstood. This is the generator?

Reply:
What do you mean? This script could be used by other people to set up a new project, right? If they use this script we need to provide all the needed the default configuration for a new project and then he will check if he need to modify the output.